### PR TITLE
Update fields in source-connectors specifications: sentry, shortio, surveymonkey, tempo, trello, us-census

### LINKED ISSUE #8370: Update fields in source-connectors specifications: sentry, shortio, stripe, surveymonkey, tempo, trello, us-census, zenloop
Add title and description properties to each field in source-connectors specifications:
source-sentry
source-shortio
source-stripe
source-surveymonkey
source-tempo
source-trello
source-us-census
source-zenloop

[EPIC] Add title and description properties to each field in connectors specifications and order input fields appropriately #8324

### PR DESCRIPTION
**Pull request by ycherniaiev**
sentry
shortio
stripe
surveymonkey
tempo
trello
us-census

### CODE REVIEW

File: airbyte-integrations/connectors/source-sentry/source_sentry/spec.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
       "project": {
         "type": "string",
         "title": "Project",
-        "description": "The slug of the project the groups belong to."
+        "description": "The Name of the Project you want to sync."
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-sentry/source_sentry/spec.json:30]:
```suggestion
        "description": "The name (slug) of the Project you want to sync."
```

File: airbyte-integrations/connectors/source-tempo/source_tempo/spec.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "properties": {
       "api_token": {
         "type": "string",
+        "title": "API token",
         "description": "Tempo API Token.",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-tempo/source_tempo/spec.json:13]: can you add a link telling where they can get this from?

**Review by sherifnada — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

**Pull request merged**